docs/maps/connect-to-ems.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[[maps-connect-to-ems]]
+== Connecting to Elastic Maps Service
+
+The **Maps** application uses https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)]
+for tiles and vector shapes of administrative boundaries.
+
+The **Maps** application proxies requests for tile layer meta data, vector shapes meta data, and vector shapes
+through the Kibana server. EMS tile requests are made directly from the browser to EMS.
+
+NOTE: The Maps application uses the Kibana server as a proxy to EMS.
```

Comment:
Maybe provide more info here? What do legacy visualizations use instead?

Using the Kibana server as a proxy to EMS is a different behavior than for legacy tile map and region map visualization.  These visualization <what do they use instead>>